ZelBack/src/services/fluxService.js

```diff
@@ -780,7 +996,7 @@ module.exports = {
   getFluxVersion,
   getFluxIP,
   getFluxZelID,
-  getFluxCruxID,
+  // getFluxCruxID, // Deprecated.
```

Comment:
Still keep for now as function is referenced in api. Soon will be most likely changed to FIO